test/functional/Java14andUp/build.xml

```diff
@@ -61,7 +61,7 @@
 		</javac>
 	</target>
 
-	<target name="dist" depends="compile,dist_functional" description="generate the distribution" >
+	<target name="dist" depends="compile" description="generate the distribution" >
```

Comment:
`dist_functional ` should not be removed. [This target](https://github.com/eclipse/openj9/blob/60f2f9454a92b0f1b7898fd7463cfb97810ee3de/test/functional/build.xml#L34) is defined in the upper level. And it is used when the `BUILD_LIST` is specifically set to subdir.

Reply:
Aha, I was fooled by `Eclipse` error message that target `dist_functional` doesn't exist in this project. Put it back now.